data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/stream/BinlogEventListener.java

```diff
@@ -281,12 +280,29 @@ void handleUpdateEvent(com.github.shyiko.mysql.binlog.event.Event event) {
         // Check if a cascade action is involved
         cascadeActionDetector.detectCascadingUpdates(event, parentTableMap, tableMetadataMap.get(data.getTableId()));
 
-        // updatedRow contains data before update as key and data after update as value
-        final List<Serializable[]> rows = data.getRows().stream()
-                .map(Map.Entry::getValue)
-                .collect(Collectors.toList());
+        final TableMetadata tableMetadata = tableMetadataMap.get(data.getTableId());
+        final List<OpenSearchBulkActions> bulkActions = new ArrayList<>();
+        final List<Serializable[]> rows = new ArrayList<>();
```

Comment:
Nit: It seems to me more readable variable name to change rows -> rowUpdates. Same applies to row -> rowUpdate.

Reply:
Thanks! Will make this change in coming PRs for RDS source.